opentech/apply/funds/forms.py

```diff
@@ -42,12 +43,12 @@ def __init__(self, *args, **kwargs):
 class UpdateReviewersForm(forms.ModelForm):
     staff_reviewers = forms.ModelMultipleChoiceField(
         queryset=User.objects.staff(),
-        widget=forms.CheckboxSelectMultiple,
+        widget=Select2MultiCheckboxesWidget,
         required=False,
     )
     reviewer_reviewers = forms.ModelMultipleChoiceField(
         queryset=User.objects.reviewers().exclude(id__in=User.objects.staff()),
-        widget=forms.CheckboxSelectMultiple,
+        widget=Select2MultiCheckboxesWidget,
```

Comment:
can you set the placeholder to say "reviewers"? Items just looks odd. Other than than good to go